packages/core/strapi/src/cli/commands/admin/list-user.ts

```diff
@@ -0,0 +1,61 @@
+import { createCommand } from 'commander';
+import CLITable from 'cli-table3';
+import chalk from 'chalk';
+import { createStrapi, compileStrapi } from '@strapi/core';
+
+import { runAction } from '../../utils/helpers';
+import type { StrapiCommand } from '../../types';
+
+/**
+ * List admin users
+ */
+const action = async () => {
+  const appContext = await compileStrapi();
+  const app = await createStrapi(appContext).load();
+
+  const list = await app.admin.services.user.findPage({
+    select: ['id', 'firstname', 'lastname', 'email', 'isActive', 'blocked'],
+    populate: ['roles'],
+    pageSize: 25,
+  });
+
+  const infoTable = new CLITable({
+    head: [
+      chalk.blue('ID'),
+      chalk.blue('Email'),
+      chalk.blue('First Name'),
+      chalk.blue('Last Name'),
+      chalk.blue('Active'),
+      chalk.blue('Blocked'),
+      chalk.blue('Roles'),
+    ],
+  });
+
+  list.results.forEach((user: any) => {
+    const roles = user.roles.map((role: any) => role.name).join(', ');
```

Comment:
Won't look the cleanest if they have a ton of roles but not sure if there is a better way to show these